### PR TITLE
Nihal branch

### PR DESCRIPTION
**Pull request by NihalMohanty**
Added PIP file and tags to some feature files.

### CODE REVIEW

File: features/pip.feature [hunk 1]
```diff
@@ -0,0 +1,20 @@
+@pip
+Feature: Checking the PIP functionality works fine
+
+As a tester of SMP
+I want to check core functionality of PIP Window
```
Comment (dalyw01) [inline features/pip.feature]: Perhaps state what PiP means first?

File: features/pip.feature [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+Background:
+  Given I am on a page with the HTML player and CP plugin installed
+  When the COOKBOOK has loaded
+  And I click CTA to begin playback
+
+# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+
+Scenario: Check basic PIP accessibility
+  And I go for the PIP options
```
Comment (dalyw01) [inline features/pip.feature]: Instead of "go" make it a bit clearer. Something like "When I click PIP icon"

File: features/pip.feature [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  And I find the pause button in the PIP window
```
Comment (dalyw01) [inline features/pip.feature]: Is this the PIP window? or the button in the control-bar?

File: features/pip.feature [hunk 4]
```diff
@@ -0,0 +1,20 @@
+@pip
+Feature: Checking the PIP functionality works fine
```
Comment (dalyw01) [inline features/pip.feature]: A negative test-case for audio needs to be added.

File: features/pip.feature [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+As a tester of SMP
+I want to check core functionality of PIP Window
+So that I verify its working as expected
+Its for chrome's PIP feature for now.
```
Comment (dalyw01) [inline features/pip.feature]: State that it is JUST for Chrome currently.

**Comment by NihalMohanty** [comment]
Done With the changes that you suggested. :)

File: features/pip.feature [hunk 6]
```diff
@@ -14,7 +14,7 @@ Background:
   And I click CTA to begin playback
 
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
-@first
+
 Scenario: Check basic PIP accessibility
```
Comment (dalyw01) [inline features/pip.feature]: Change accessibility to functionality.

The word accessibility suggests that it's for users who are impaired.

File: features/pip.feature [hunk 7]
```diff
@@ -33,9 +33,17 @@ Examples:
   | page                 | type  | device  |
   | https://is.gd/hayuwa | audio | desktop |
 
-  # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
 
 Scenario: Full screen goes off when you go for PIP mode
```
Comment (dalyw01) [inline features/pip.feature]: What do you mean by "goes off" ?

**Comment by NihalMohanty** [comment]
Resolved the comments. :)

**Pull request merged**